ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -378,32 +391,78 @@ class ServicesHomeScreen extends React.Component<Props, State> {
 const mapStateToProps = (state: GlobalState) => {
   const { services } = state.entities;
 
+  const potUserMetadata = userMetadataSelector(state);
+  const userMetadata = pot.getOrElse(potUserMetadata, undefined);
```

Comment:
since _**potUserMetadata**_ is a _pot_ you don't know if the value is available or not at this point.
In this way, you could have an undefined value because the value is in loading state.
Could you check if this is an issue?

Another point of discussion is the value returned by [fetchUserMetadata](fetchUserMetadata). Maybe @francescopersico  could help us: when the backend responses with **_status===204_** (no content) an error will be returned. In this way, the _userMetaData_ will never be updated cause the _pot_ is in error status. I guess, in that specific case, we should provide an empty _userMetaData_ object